src/app/core/service/api.service.ts

```diff
@@ -111,24 +110,11 @@ export class ApiService implements HttpInterceptor {
     this.router.navigate([APP_ROUTES.AUTH_LOGIN]);
   }
 
-  private setHeaders(path: string, contentType = ''): HttpHeaders {
+  protected setHeaders(path: string, setAuthToken: any, contentType = ''): HttpHeaders {
```

Comment:
1. Introducing new params shouldn't be middle, not a good practice
2. path is not used